kafka-client/src/main/java/dev/responsive/kafka/internal/utils/StoreUtil.java

```diff
@@ -92,6 +93,17 @@ public static long durationToMillis(final Duration duration, final String parame
     }
   }
 
+  public static String streamThreadId() {
```

Comment:
Looks like we already have `ResponsiveMetrics#computeThreadId` - may want to consider reusing that (it also addresses Global store threads)